pkgs/by-name/ew/eww/package.nix

```diff
@@ -7,24 +7,30 @@
 , librsvg
 , gtk-layer-shell
 , stdenv
+, libdbusmenu-gtk3
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "eww";
-  version = "0.5.0";
+  version = "0.6.0";
 
   src = fetchFromGitHub {
     owner = "elkowar";
     repo = "eww";
-    rev = "v${version}";
-    hash = "sha256-HBBz1NDtj2TnDK5ghDLRrAOwHXDZlzclvVscYnmKGck=";
+    rev = "refs/tags/v${version}";
```

Comment:
pls remove `refs/tags/` because it already has the tags from the eww repository.

Reply:
There's really no reason to remove it; this ensures that we're explicitly fetching a tag and not an identically-named branch, or something else that can confuse an update script or update bot.

Reply:
with `refs/tags` will work but also without `refs/tags` the same, but as you saw the previous eww commits did not have that `refs/tags` and it works fine. 
so please. if your reason with `refs/tags` is ok.